aces/Loris
# [Config] Add password algorithm selector

## Brief summary of changes

- In CCNA, we use PASSWORD_ARGON2I rather than PASSWORD_DEFAULT
- This PR adds an option to change which php password algorithm to use so that projects do not need an override in-order to change it.
- It defaults to PASSWORD_DEFAULT which is what it was set to before.
![image](https://github.com/user-attachments/assets/3793805f-356a-46a4-b695-19be75947151)


#### Testing instructions (if applicable)

1. Try changing the password hash in config, and then setting a new password.
2. See that you can still login
